archeryutils/handicaps/handicap_equations.py

```diff
@@ -355,15 +356,126 @@ def sigma_r(
     return sig_r
 
 
+def s_bar(
+        target_specs: dict[float, int],
+        arw_rad: float,
+        sig_r: FloatArray
+    ) -> FloatArray:
+    """Calculate expected score directly from target ring sizes.
+
+    Parameters
+    ----------
+    target_specs : dict[float, int]
+        Mapping of target ring *diameters* in [metres], to points scored
+    arw_d : float
+        arrow diameter in [metres]
+    sig_r : float
+        standard deviation of group size [metres]
+
+    Returns
+    -------
+    s_bar : float
+        expected average score per arrow
+
+    Notes
+    -----
+    May differ from previous implementions due to floating point errors in deriving
+    ring radiuses directly from diameters, rather than multiplying by ring count and
+    dividing by overall diameter
+    Eg 0.12 / 2 != 3*0.4 / 20
+
+    Examples
+    --------
+    >>> #WA 18m compound triple spot
+    >>> specs = {0.02: 10, 0.08: 9, 0.12: 8, 0.16: 7, 0.2: 6}
+    >>> _sbar(specs, 9.3e-3, 0.04)
```

Comment:
I agree good docs are needed to explain how a face is described by `spec`, possibly as an addendum to the `Target` class user docs rather than in the code...

Reply:
Makes sense to shift the Target specific parts into the class and document there

Reply:
I know this is explained in the 'getting started' docs, but I think a clear description of what a `face_spec` is in the docstrings so that it appears in the API docs would be good too.
Not 100% sure where is best for this, perhaps under the `face_spec` parameter, but it could be a bit long, so maybe in the extended description or the `from_face_spec` description...?